internal/daemonset/daemonset.go

```diff
@@ -310,3 +322,44 @@ func OverrideLabels(labels, overrides map[string]string) map[string]string {
 
 	return labels
 }
+
+func MakeLoadCommand(spec ootov1alpha1.ModprobeSpec) []string {
+	loadCommand := []string{"modprobe"}
+
+	if ra := spec.RawArgs; ra != nil && len(ra.Load) > 0 {
+		return append(loadCommand, ra.Load...)
+	}
+
+	if a := spec.Args; a != nil && len(a.Load) > 0 {
+		loadCommand = append(loadCommand, a.Load...)
+	} else {
+		loadCommand = append(loadCommand, "-v")
+	}
+
+	if dirName := spec.DirName; dirName != "" {
+		loadCommand = append(loadCommand, "-d", dirName)
+	}
+
+	loadCommand = append(loadCommand, spec.ModuleName)
+	return append(loadCommand, spec.Parameters...)
+}
+
+func MakeUnloadCommand(spec ootov1alpha1.ModprobeSpec) []string {
+	unloadCommand := []string{"modprobe"}
+
+	if ra := spec.RawArgs; ra != nil && len(ra.Load) > 0 {
+		return append(unloadCommand, ra.Load...)
+	}
+
+	if a := spec.Args; a != nil && len(a.Unload) > 0 {
+		unloadCommand = append(unloadCommand, a.Unload...)
+	} else {
+		unloadCommand = append(unloadCommand, "-rv")
```

Comment:
I think we should always add `"-rv"`. Are we expecting the user to add the `-r` on each module?

Reply:
We are expecting users not to use `Args`. But if they do, they should be able to do everything they want - including overwriting the default `-rv`.

Reply:
Now that I understand better the difference between `args`, `rawArgs` and `parameters` it make more sense.